dev/io.openliberty.microprofile.telemetry.1.1.internal/src/io/openliberty/microprofile/telemetry11/internal/rest/TelemetryServletFilter.java

```diff
@@ -312,17 +309,6 @@ public List<String> getHttpResponseHeader(final ServletRequest request, final Se
 
     }
 
-    private HashMap<String, String> getTelemetryProperties() {
-        HashMap<String, String> telemetryProperties = new HashMap<>();
-        for (String propertyName : config.getPropertyNames()) {
-            if (propertyName.startsWith("otel") || propertyName.startsWith("OTEL")) {
-                config.getOptionalValue(propertyName.toLowerCase().replace('_', '.'), String.class).ifPresent(
-                    value -> telemetryProperties.put(propertyName.toLowerCase().replace('_', '.'), value));
-            }
-        }
-        return telemetryProperties;
-    }
-
```

Comment:
The method below this one can be removed as well I think.

Reply:
I also think these changes apply to the other version of `TelemetryServletFilter` that we have.

Reply:
Done